use-cases/error-monitoring/README.md

```diff
@@ -56,7 +56,12 @@ EmailId=user@domain.com
 Schedule=rate(1 minute)
 ```
 
-### 2. Configure Sample Solution App's IAM user
+### 2. Set region code and marketplace id
+You have to set region code and marketplace id that matches your account in the code. By default it is set to US.
+If you want to change the values go to [ErrorMonitoringHandler.java](https://github.com/amzn/selling-partner-api-samples/blob/main/use-cases/error-monitoring/code/java/src/main/java/lambda/ErrorMonitoringHandler.java) and enter the proper data.
```

Comment:
Here you can point to the specific lines: https://github.com/amzn/selling-partner-api-samples/blob/main/use-cases/error-monitoring/code/java/src/main/java/lambda/ErrorMonitoringHandler.java#L28-L31